assets/docs/configuration/transformations/custom-scripts/examples/js-snowplow-config-example.hcl

```diff
@@ -7,6 +7,10 @@ transform {
     # For this example, we use an environment variable, to facilitate unit tests. A hardcoded value may also be provided (eg. "/tmp/myscript.js")
     script_path = env.JS_SNOWPLOW_SCRIPT_PATH
 
-    snowplow_mode       = true # Snowplow mode enabled - this transforms the tsv to an object on input
+    snowplow_mode = true # Snowplow mode enabled - this transforms the tsv to an object on input
```

Comment:
Let's remove old `snowplow_mode` from this example - it's counterintuitive to configure both, so I think the example we provide in docs should just configure the new one :)